app/models/location.rb

```diff
@@ -1,6 +1,8 @@
 class Location < ActiveRecord::Base
   belongs_to :created_by, class_name: 'User', foreign_key: :creator_id
   belongs_to :updated_by, class_name: 'User', foreign_key: :updater_id
+  has_many :cycle_counts
+  has_many :pallets, through: :cycle_counts
```

Comment:
Would this be for a history of all the pallets that were believed to appear in that location?